src/main/java/com/karimbouchareb/chromatographysimulator/ui/uiUtility.java

```diff
@@ -0,0 +1,27 @@
+package com.karimbouchareb.chromatographysimulator.ui;
+
+import com.karimbouchareb.chromatographysimulator.ChromatographySimulatorApp;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.stage.Screen;
+
+import java.net.URL;
+
+public class uiUtility {
```

Comment:
Need to name class according to convention, which is each word is capitalized. Maybe we can just rename this to something else like ScreenUtil?